static/src/components/fieldsconfig/consensus_config.vue

```diff
@@ -157,7 +149,37 @@ export default {
             if (res.ok) {
                 const data = await res.json();
                 window.pybossaNotify(data.flash, true, data.status);
-                this.isDefined = this.threshold !== 0;
+                this.updateConsensusConfig({
```

Comment:
This looks good. I believe you may be able to [omit](http://www.benmvp.com/learning-es6-enhanced-object-literals/#tldr) the values in these object literals, since they have the same naming as the keys. For example:

```js
this.updateConsensusConfig({ threshold, maxRetries, redundancyDelta });
```

Reply:
I can't tell, but i don't think `threshold`, `maxRetries`, `redundancyDelta` are defined in this scope, only `this.threshold`, `this.maxRetries`, `this.redundancyDelta`...so I don't think this syntax will work here.